nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/processors/hive/PutHiveStreaming.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.avro.generic.GenericDatumWriter;
 import org.apache.avro.generic.GenericRecord;
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
```

Comment:
Minor CheckStyle violation (unused import), I will remove before merging your PR. For the future, you can run "mvn clean install -Pcontrib-check" from the nifi-nar-bundles/nifi-hive-bundle to catch these errors (and any compilation or unit test errors).